Release/tests/functional/websockets/client/send_msg_tests.cpp

```diff
@@ -456,6 +471,25 @@ TEST_FIXTURE(uri_address, send_stream_binary_msg_no_length)
     client.close().wait();
 }
 
+// Send an unsolicited pong message to the server
+TEST_FIXTURE(uri_address, send_pong_msg)
```

Comment:
These tests will fail on WinRT since our WinRT websocket implementation does not support ping/pong messages. We should disable these tests on winrt (#if !defined(__cplusplus_winrt)). 
Alternatively, I can also disable these tests for WinRT once we accept the PR.